doc/syntax.org

```diff
@@ -0,0 +1,43 @@
+#+TITLE:   Markup Syntax Guide for Pancacke(d)
+#+AUTHOR:  Scott Wang
+#+STARTUP: align indent
+#+OPTIONS: H:5
+
+
+* Overview
```

Comment:
Then we can declare it in the `Headlines` section or `Keywords and Reserved Words` section. This document (the document file this PR modifies) itself is the language specification.